api/proto/disperser/disperser.proto

```diff
@@ -115,17 +127,39 @@ message BlobStatusReply {
 	BlobInfo info = 2;
 }
 
-// RetrieveBlobRequest contains parameters to retrieve the blob.
-message RetrieveBlobRequest {
+// GetBlobRequest contains parameters to retrieve the blob.
+message GetBlobRequest {
 	bytes batch_header_hash = 1;
 	uint32 blob_index = 2;
 }
 
-// RetrieveBlobReply contains the retrieved blob data
-message RetrieveBlobReply {
+// GetBlobReply contains the retrieved blob data
+message GetBlobReply {
```

Comment:
I wonder if we should update this interface to also return the header.

Reply:
It would be this: https://github.com/Layr-Labs/eigenda/blob/dbeaf12c3a59c1d5c3b49ad7a0d59d07576fe9ca/api/proto/disperser/disperser.proto#L173

But there's some standardization we should do here. I'd like to get started on this next sprint. https://www.notion.so/eigen-labs/Unified-blob-addressal-in-EigenDA-ae426c610f9e4e61b2e46eec76f7ca4d?d=f09f902f4d0743a7abef61faae0e2045#d392986bccbc43f9b6f8bcf6ace29255

Reply:
updated to support lookup via both mechanisms